ios-xcframework/BuildPhases/set_current_project_version.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash -eu
+
+target_plist="$TARGET_BUILD_DIR/$INFOPLIST_PATH"
+dsym_plist="$DWARF_DSYM_FOLDER_PATH/$DWARF_DSYM_FILE_NAME/Contents/Info.plist"
+
+version=$("$(dirname "$0")"/get_gutenberg_version)
+
+for plist in "$target_plist" "$dsym_plist"; do
+  if [ -f "$plist" ]; then
+    /usr/libexec/PlistBuddy -c "Set :CFBundleVersion $version" "$plist"
+    /usr/libexec/PlistBuddy -c "Set :CFBundleShortVersionString $version" "$plist"
+  fi
+done
```

Comment:
This script is not used anymore, I think?

Reply:
Thanks! I had forgotten about that one.

Reply:
Done via 6d5a033d89e51b5c84b29816743b74258aa92fbd